src/ol_infrastructure/applications/concourse/__main__.py

```diff
@@ -40,6 +42,62 @@
 from ol_infrastructure.lib.stack_defaults import defaults
 from ol_infrastructure.lib.vault import setup_vault_provider
 
+
+def build_worker_user_data(concourse_team, concourse_tags, consul_dc):
```

Comment:
It would be good to add some type information to the function signature. Something like:
`def build_worker_user_data(concourse_team: str, concourse_tags: List[str], consul_dc: str) -> str:`

Reply:
👍